pkg/validate/apparmor_linux.go

```diff
@@ -82,37 +82,64 @@ var _ = framework.KubeDescribe("AppArmor", func() {
 			})
 
 			It("should fail with an unloaded profile", func() {
-				profile := apparmorProfileNamePrefix + "non-existent-profile"
+				profile := &runtimeapi.LinuxContainerSecurityContext{
+					ApparmorProfile: apparmorProfileNamePrefix + "non-existent-profile",
+				}
 				containerID := createContainerWithAppArmor(rc, ic, sandboxID, sandboxConfig, profile, false)
 				Expect(containerID).To(BeEmpty())
 			})
 
 			It("should enforce a profile blocking writes", func() {
-				profile := apparmorProfileNamePrefix + "cri-validate-apparmor-test-deny-write"
+				profile := &runtimeapi.LinuxContainerSecurityContext{
+					ApparmorProfile: apparmorProfileNamePrefix + "cri-validate-apparmor-test-deny-write",
+				}
 				containerID := createContainerWithAppArmor(rc, ic, sandboxID, sandboxConfig, profile, true)
 				checkContainerApparmor(rc, containerID, false)
 			})
 
 			It("should enforce a permissive profile", func() {
-				profile := apparmorProfileNamePrefix + "cri-validate-apparmor-test-audit-write"
+				profile := &runtimeapi.LinuxContainerSecurityContext{
+					ApparmorProfile: apparmorProfileNamePrefix + "cri-validate-apparmor-test-audit-write",
+				}
 				containerID := createContainerWithAppArmor(rc, ic, sandboxID, sandboxConfig, profile, true)
 				checkContainerApparmor(rc, containerID, true)
 			})
+
+			It("should work with another field", func() {
```

Comment:
please instead of calling it "another", be more specific.

Also please add a note to already existing tests to say that those are testing "deprecated" fields

Reply:
yeah, sorry, I saw your previous comments. I've rebased the branch and am going to change the naming, I just didn't push all my changes. 🙏